custom/ewsghana/utils.py

```diff
@@ -266,17 +267,11 @@ def receipts(self):
         ]
 
 
-def get_reporting_types(domain):
-    return [
-        location_type for location_type in Domain.get_by_name(domain).location_types
-        if not location_type.administrative
-    ]
-
-
 def can_receive_email(user, verified_number):
     return user.email and verified_number.backend_id and verified_number.backend_id == 'MOBILE_BACKEND_TWILIO'
 
 
+@quickcache(['domain'])
 def get_country_id(domain):
```

Comment:
This is a small lookup, but it looks like it's called about 5 times per request, and it's very easy to cache.